.eslintrc.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  "env": {
+    "browser": true,
+    "commonjs": true,
+    "es6": true,
+    "node": true
+  },
+  "extends": "eslint-config-digitalbazaar",
+  "parserOptions": {
+    "ecmaVersion": 2018
+  },
+  "rules": {
```

Comment:
Are these fixed in the latest release of `eslint-config-digitalbazaar`?

Reply:
Updated.